include/z64scene.h

```diff
@@ -453,6 +453,13 @@ typedef enum {
     /* 53 */ SDC_MAX
 } SceneDrawConfig;
 
+// R_SCENE_CAM_TYPE, first arg to SCENE_CMD_MISC_SETTINGS()
```

Comment:
I would put the defines near the `SCmdMiscSettings` struct but it's not a big deal, headers are messy